src/simplewallet/simplewallet.cpp

```diff
@@ -1462,6 +1464,72 @@ bool simple_wallet::handle_command_line(const boost::program_options::variables_
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::prompt_for_new_node()
+{
+    bool connect = false;
+    std::string node_url;
+
+    message_writer() << tr("Do you want to connect to a remote node?");
+    std::string confirm = command_line::input_line(tr("(Y/Yes/N/No): "));
+    if(std::cin.eof())
+    {
+      LOG_ERROR("Unexpected std::cin.eof() - Exited simple_wallet::prompt_for_new_node()");
+      return false;
+    }
+    connect = command_line::is_yes(confirm);
+
+    if (connect) {
+        message_writer() << tr("Please enter remote node: ");
+        node_url = command_line::input_line(tr("http(s)://[domain]:[port] (default port 18081): "));
+
+        boost::regex rgx("^(.*)://([A-Za-z0-9\\-\\.]+)(:[0-9]+)?(.*)");
+        boost::cmatch match;
+        if (boost::regex_match(node_url.c_str(), match, rgx)) {
+            // If no port has been entered, use the default
+            if (!match[3].length()) {
+                node_url = match[1] + std::string("://") + match[2] + std::string(":18081");
+            }
+        }
+
+        if (match.size() >= 2) {
+            if (!std::string("https").compare(match[1])) {
+                message_writer() << tr("Attention: Nodes not supporting 'https://' will transmit data unencrypted.\n"
+                                       "Do you still want to connect to this node?");
```

Comment:
That's all of them.

Reply:
Even with `https://`? I would've assumed not.

Reply:
Those do not exist. The code does not support SSL for RPC communication. If you saw SSL support in the code, it's for mymonero style servers. There *might* be SSL RPC support later, but there is none now AFAIK.

Reply:
Ok, removed.